### PR TITLE
[Done] Translation of faq-build.md ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/docs/faq-build.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 ---
 id: faq-build
-title: Babel, JSX, and Build Steps
+title: Babel, JSX و خطوات بناء التطبيقات
```
Comment (mohshbool) [inline content/docs/faq-build.md]:
```suggestion
title: Babel و JSX و خطوات بناء التطبيقات
```

File: content/docs/faq-build.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
 permalink: docs/faq-build.html
 layout: docs
 category: FAQ
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### هل أحتاج إلى استخدام JSX مع React؟ {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+لا! انتقل إلى مستند ["استخدام React بدون JSX"](/docs/react-without-jsx.html) لتعلّم المزيد.
```
Comment (mohshbool) [inline content/docs/faq-build.md]:
```suggestion
لا! انظر إلى مستند ["استخدام React بدون JSX"](/docs/react-without-jsx.html) لتعلّم المزيد.
```
```suggestion
لا! تفحّص مستند ["استخدام React بدون JSX"](/docs/react-without-jsx.html) لتعلّم المزيد.
```

File: content/docs/faq-build.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### هل أحتاج إلى استخدام ES6 (+) مع React؟ {#do-i-need-to-use-es6--with-react}
 
-No! Check out ["React Without ES6"](/docs/react-without-es6.html) to learn more.
+لا! انتقل إلى مستند ["استخدام React بدون ES6"](/docs/react-without-es6.html) لتعلّم المزيد.
```
Comment (mohshbool) [inline content/docs/faq-build.md]: same as the above

File: content/docs/faq-build.md [hunk 4]
```diff
@@ -1,32 +1,32 @@
 ---
 id: faq-build
-title: Babel, JSX, and Build Steps
+title: Babel و JSX و خطوات بناء التطبيقات
```
Comment (Fcmam5) [inline content/docs/faq-build.md]:
```suggestion
title: Babel و JSX وخطوات بناء التطبيقات
```

Comment (3imed-jaberi) [reply content/docs/faq-build.md]: Is it possible to clarify more !!

Comment (mohshbool) [reply content/docs/faq-build.md]: I think he's just suggesting that we remove an unnecessary whitespace 😅

Comment (3imed-jaberi) [reply content/docs/faq-build.md]: are you serious 🤣  !

Comment (Fcmam5) [reply content/docs/faq-build.md]: @3imed-jaberi Yes, as Mohammad said, the whitespace is unnecessary, sorry I should mention it.
This is a common mistake and Hsoub's writing guidelines also mention it, I just can't find the link where I read it.
But you can check this [link](https://www.bbc.co.uk/academy/ar/articles/art20130702112134021) as well.

**Pull request merged**